azure-pipelines.yml

```diff
@@ -0,0 +1,66 @@
+# ASP.NET
+# trigger on main pull
+trigger:
+  branches:
+    include:
+    - main
+# initial build
+stages:
+- stage: build
+  jobs:
+  - job: build
+    variables:
+        imageName: 'fakebookposts'
+
+    steps:
+    - task: Docker@2
+      displayName: Build Docker Image
+      inputs:
+        repository: $(imageName)
+        command: 'build'
+        Dockerfile: 'Fakebook.Posts/Dockerfile'
+# sonar cloud
+- stage: sonarCloud
```

Comment:
this stage is not dependent on the previous one so use dependsOn like the other merged pipelines should be